### PR TITLE
Creates a new feature spec to test clients

### PR DESCRIPTION
**Pull request by edgibbs**
## Description

Handles testing of client pages and also handles perry session logins across tests.  May want to change to logout down the road, but reasonable for now.

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: spec/acceptance/client_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    fill_in('First Name', with: '')
+    click_button('Save')
+    expect(page).to have_content 'Add Child/Youth'
+
+    fill_in('First Name', with: first_name)
+    fill_in('Last Name', with: last_name)
+    page.find('#dob').set(date_of_birth)
+    fill_in('Case Number', with: case_number)
+    find('.jss96').click
```
Comment (denys-davydov) [inline spec/acceptance/client_page_spec.rb]: This '96' suffix looks like a generated number, so any change on the page will lead to the number to be changed and test to become broken. Can we use a different selector if it really works that way?

Comment (edgibbs) [reply spec/acceptance/client_page_spec.rb]: Good catch, I'll take another quick pass at it, the issue is related to how the component is rendered and at some point I just moved on, but there should be some way to put a better marker on it.

Comment (edgibbs) [reply spec/acceptance/client_page_spec.rb]: OK, updated to not rely on a generated class name.

File: spec/acceptance/client_page_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```
Comment (jonbwalker) [inline spec/acceptance/client_page_spec.rb]: Does something use frozen_string_literal? I've seen this in some of our other files.

Comment (edgibbs) [reply spec/acceptance/client_page_spec.rb]: Oh, it's a future version of ruby thing for performance enhancements:'

https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html

**Pull request merged**